swift/apple/FirezoneKit/Sources/FirezoneKit/AuthClient/AuthClient.swift

```diff
@@ -43,10 +43,11 @@ private final class WebAuthenticationSession: NSObject,
   @MainActor
   func signIn(_ host: URL) async throws -> Token {
     try await withCheckedThrowingContinuation { continuation in
-      let callbackURLScheme = "firezone-fd0020211111"
+      let callbackURLScheme = "firezone"
       let session = ASWebAuthenticationSession(
-        url: host.appendingPathComponent("auth")
-          .appendingQueryItem(URLQueryItem(name: "dest", value: "\(callbackURLScheme)://auth")),
+        url: host.appendingPathComponent("sign_in")
```

Comment:
@jamilbk I should add the team-id we get from settings here, before the `/sign_in`, right? (Will do that in a subsequent PR.)

Reply:
Yep yep! All sign in URLs will be scoped by the team ID